src/SearchInput/SearchInput.tsx

```diff
@@ -130,11 +158,22 @@ const Container = styled.div<IContainer>`
   background: ${theme.colors.white};
 `
 
+const InnerContainer = styled.div<IInnerContainer>`
+  ${(p) =>
+    p.hasBorder
+      ? `
+    border: 1px solid ${theme.colors.grey3};
+    border-radius: 8px;
+    padding: 16px;
+  `
+      : ``}
+`
+
```

Comment:
shouldn't this has position relative otherwise the icon will be positioned relative to the parent

Reply:
This can be shortened by using `&&` instead of a `?` => `p.hasBorder && border: 1px solid ${theme.colors.grey3};` etc